pom.xml

```diff
@@ -159,6 +159,16 @@
       <artifactId>mockito-core</artifactId>
       <version>2.7.21</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.jclouds.api</groupId>
+      <artifactId>chef</artifactId>
+      <version>2.0.3</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
Please, no :sweat: you are only going to bring a conflicting version fight :laughing:

Can you rewrite the code without this version of guava.
If this version guava is needed because of `org.apache.jclouds.api:chef` I suggest removing this feature.